src/components/organisms/JoinVenueModal/JoinVenueModal.tsx

```diff
@@ -0,0 +1,57 @@
+import React, { FC, Fragment, useCallback } from "react";
+import { Modal } from "react-bootstrap";
+import { useHistory } from "react-router-dom";
+
+import { AnyVenue } from "types/Firestore";
+import { venueEntranceUrl, venueInsideUrl } from "utils/url";
+
+import "./JoinVenueModal.scss";
+
+interface JoinVenueModal {
+  venueId?: string;
+  venue: AnyVenue;
+  show: boolean;
+  isLoggedIn: boolean;
+  onHide: () => void;
+}
+
+export const JoinVenueModal: FC<JoinVenueModal> = ({
+  show,
+  venueId,
+  venue,
+  isLoggedIn,
+  onHide,
+}) => {
+  const history = useHistory();
+
+  const joinVenue = useCallback(() => {
+    if (venueId) {
+      const venueEntrance = venue?.entrance && venue.entrance.length;
```

Comment:
```suggestion
      const hasVenueEntrance = venue?.entrance?.length ?? 0 > 0;
```

Reply:
How about `isTruthy(venue?.entrance?.length)` ?

Reply:
Yeah, that’d also work in this case.

I have a growing preference of explicitly checking ‘length > 0’ rather than ‘!foo.length’ or similar. But in this case given it also could be ‘undefined’, the helper you suggest makes sense.